go-controller/pkg/util/ovs.go

```diff
@@ -25,6 +29,112 @@ func PathExist(path string) bool {
 	return true
 }
 
+func runningPlatform() (string, error) {
+	fileContents, err := ioutil.ReadFile(osRelease)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse file %s (%v)", osRelease, err)
+	}
+
+	var platform string
+	ss := strings.Split(string(fileContents), "\n")
```

Comment:
At some point we should use a config parser like this: https://github.com/BurntSushi/toml